mindjunkies/accounts/tests.py

```
@@ -0,0 +1,56 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from home.views import home
```

Ruff (0.8.2)

`home.views.home` imported but unused

Remove unused import: `home.views.home`

(F401)